get fails for gdal 2.0

[documentation states only gdal 1.11 has been tested; this is just a warning note to users of *current* 2.0]
June 2015 GDAL/OGR 2.0.0 release
```
# github.com/lukeroth/gdal
lukeroth/gdal/gdal.go: In function ‘_cgo_6a4191d94aa1_Cfunc_GDALGetDefaultHistogram’:
lukeroth/gdal/gdal.go:760:2: warning: ‘GDALGetDefaultHistogram’ is deprecated [-Wdeprecated-declarations]
# github.com/lukeroth/gdal
lukeroth/gdal/ogr.go:1238: cannot use C.long(fid) (type C.long) as type C.GIntBig in argument to _Cfunc_OGR_F_SetFID
lukeroth/gdal/ogr.go:1329: cannot use C.long(index) (type C.long) as type C.GIntBig in argument to _Cfunc_OGR_L_SetNextByIndex
lukeroth/gdal/ogr.go:1334: cannot use C.long(index) (type C.long) as type C.GIntBig in argument to _Cfunc_OGR_L_GetFeature
lukeroth/gdal/ogr.go:1350: cannot use C.long(index) (type C.long) as type C.GIntBig in argument to _Cfunc_OGR_L_DeleteFeature
```

https://svn.osgeo.org/gdal/branches/2.0/gdal/MIGRATION_GUIDE.TXT
- D) RFC 31: OGR 64bit Integer Fields and FIDs
  - http://trac.osgeo.org/gdal/wiki/rfc31_ogr_64
- J) RFC 57: 64-bit bucket count for histograms
  - https://trac.osgeo.org/gdal/wiki/rfc57_histogram_64bit_count

--- a/ogr.go
+++ b/ogr.go
@@ -1249,7 +1249,7 @@ func (feature Feature) FID() int {
 
 // Set feature identifier
 func (feature Feature) SetFID(fid int) error {
-	return C.OGR_F_SetFID(feature.cval, C.long(fid)).Err()
+	return C.OGR_F_SetFID(feature.cval, C.GIntBig(fid)).Err()
 }
 
 // Unimplemented: DumpReadable
@@ -1340,12 +1340,12 @@ func (layer Layer) NextFeature() Feature {
 
 // Move read cursor to the provided index
 func (layer Layer) SetNextByIndex(index int) error {
-	return C.OGR_L_SetNextByIndex(layer.cval, C.long(index)).Err()
+	return C.OGR_L_SetNextByIndex(layer.cval, C.GIntBig(index)).Err()
 }
 
 // Fetch a feature by its index
 func (layer Layer) Feature(index int) Feature {
-	feature := C.OGR_L_GetFeature(layer.cval, C.long(index))
+	feature := C.OGR_L_GetFeature(layer.cval, C.GIntBig(index))
 	return Feature{feature}
 }
 
@@ -1361,7 +1361,7 @@ func (layer Layer) Create(feature Feature) error {
 
 // Delete indicated feature from layer
 func (layer Layer) Delete(index int) error {
-	return C.OGR_L_DeleteFeature(layer.cval, C.long(index)).Err()
+	return C.OGR_L_DeleteFeature(layer.cval, C.GIntBig(index)).Err()
 }
 
 // Fetch the schema information for this layer

